app/assets/javascripts/angular/controllers/desktop/news.js

```diff
@@ -0,0 +1,52 @@
+(function() {
```

Comment:
Many controllers are broken up into their own things now to keep files smaller and more concise. This is the news display on mobile, complete with animations.